### ISSUE TITLE
Custom RSS Feed Tagging

### ISSUE THREAD

**Issue by coma64** [omnivore-app/omnivore#2923]
Hi there,

I just started using omnivore and I'd like to replace my RSS feed reader with it. Unfortunately though, omnivore doesn't have RSS feed folders, like [feedly](https://blog.feedly.com/feeds-and-folders/) does, which is a must have feature for me.

I think in omnivore it could be implemented quite easily by giving each article from a feed one or more customisable tags.

I wouldn't mind implementing this myself. If that's okay please give me a heads up

**Comment by antoinerey** [comment]
I just found out about rules: https://docs.omnivore.app/using/rules.html. I believe it solves your need. By creating a rule that targets your RSS feed, you could add a custom label to all its articles.

Mine looks like this:

<img width="934" alt="CleanShot 2023-10-12 at 23 34 06@2x" src="https://github.com/omnivore-app/omnivore/assets/3336873/09ccf781-72c5-4868-a8ba-f6299feebcbc">

**Comment by coma64** [comment]
You're right that does solve my problem although it's quite cumbersome to tag feeds that way. I think it would be a great addition to be able to create a rule for a feed directly from the feed page.

also, I'm not seeing the rules page in my navbar (but I can open it with the url) so I suppose it's still in some kind of beta

**Comment by antoinerey** [comment]
<img width="707" alt="CleanShot 2023-10-17 at 11 01 49@2x" src="https://github.com/omnivore-app/omnivore/assets/3336873/16a3b520-8e5d-445e-839e-da8177235e0e">

Yup. Looks like it's still in beta.

**Comment by hkunzhe** [comment]
Any updates?

**Comment by jacksonh** [comment]
@hkunzhe you can achieve this with rules. In iOS you can do it from the subscriptions page, we still haven't added it to the web, but it is possible now with rules.

**Comment by Podginator** [comment]
Closing due to age, and the self-hosted nature of Omnivore. Done as a batch action. If this issue still exists, please reopen it.
